tests/__init__.py

```
@@ -355,10 +355,20 @@ def loadDocstringsFromModule(module: types.ModuleType) -> TestSuite:
 except Exception:  # pragma: no branch
 	_LOGGER.warning("Error loading optional debug tests", exc_info=True)
 
+try:
+	EXTRA_TESTS["coverage"].append(loadDocstringsFromModule(__module__))
+	EXTRA_TESTS["coverage"].append(loadDocstringsFromModule(context))
+	EXTRA_TESTS["coverage"].append(loadDocstringsFromModule("tests.MulticastUDPClient"))
+except (ImportError, ValueError) as _cause:  # pragma: no branch
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)